.github/workflows/tests.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       matrix:
         os: ["macos-latest", "ubuntu-20.04", "windows-latest"]
-        python-version: ["3.6", "3.7", "3.8", "3.9"]
+        python-version: ["3.7", "3.8", "3.9"]
```

Comment:
3.10?

Reply:
Could do. I thought we checked this in an earlier PR. @micahfolsom can you add to try in the CI?